src/util-inl.h

```diff
@@ -216,6 +217,8 @@ inline v8::Local<v8::String> OneByteString(v8::Isolate* isolate,
 void SwapBytes16(char* data, size_t nbytes) {
   CHECK_EQ(nbytes % 2, 0);
 
+  ::swap_simd(data, &nbytes, 16);
```

Comment:
Again, I might be missing something. Why do these calls fall through to the old implementation? Should it not return here?

Reply:
swap_simd only process n*16 bytes in this case. When swap_simd returns, the data and nbytes have been updated and the working flow goes into below legacy code to process remained bytes. (nbytes-n*16) bytes